test/test_convolve_scanners.py

```diff
@@ -39,10 +52,27 @@ def transform_a(x: any):
         """
         return 4 * x + 0.025
 
+    def transform_ones_like(x: any):
```

Comment:
Why is this function needed? Can't we just replace its use with `jnp.ones_like(x)`?

Reply:
It's not! It was constructed without much thought to replace a lambda expression. 
replaced with `jnp.ones_like` in https://github.com/CDCgov/PyRenew/pull/440/commits/cc61d0b2f3a973109808ffa2c222aaeb319be3c3

Reply:
Agree have suggested below.